py/client/pydeephaven/session.py

```diff
@@ -60,7 +115,15 @@ def __init__(self, host: str = None, port: int = None, never_timeout: bool = Tru
             self.port = int(os.environ.get("DH_PORT", 10000))
 
         self.is_connected = False
-        self.session_token = None
+
+        if auth_type == "Anonymous":
+            self._auth_token = auth_type
+        elif auth_type == "Basic":
+            auth_token_base64 = base64.b64encode(auth_token.encode("ascii")).decode("ascii")
```

Comment:
@niloc132 If the Go client should be split up into two args, it probably also needs to base64 encode...right?

Reply:
Only the `Basic` type is expected/required to be base64 encoded, this isn't necessarily required for other formats. For example, if you base64 the already-hex encoded uuids that are used for bearer tokens, or the already-base64 encoded openId connect payloads, the server won't be able to read them.